plyer/platforms/android/devicename.py

```diff
@@ -15,10 +14,16 @@ class AndroidDeviceName(DeviceName):
     '''
 
     def _get_device_name(self):
-        return Secure.getString(
-            activity.getContentResolver(),
-            Secure.DEVICE_NAME
-        )
+        """
+        Method to get the device name in an android environment.
+
+        Changed the implementation from 'android.provider.Settings.Global' to
+        'android.os.Build' becuase 'android.provider.Settings.Global' was
```

Comment:
```suggestion
        'android.os.Build' because 'android.provider.Settings.Global' was
```

Reply:
Thank you for your comment.
There is no change in the result, its the same.